src/lxc/attach.c

```diff
@@ -314,6 +314,9 @@ static int lxc_attach_drop_privs(struct lxc_proc_context_info *ctx)
 	int cap, last_cap;
 
 	last_cap = lxc_caps_last_cap();
+	if (last_cap < 0 || last_cap > INT_MAX)
```

Comment:
`las_cap` can't be greater than `INT_MAX` since it is an int itself. :) You could make `last_cap` a `long int` and perform that check.

Reply:
@brauner Yes, you're right. I'll check it again. Thanks for your review. :)